api/delete-full.md

```diff
@@ -1,10 +1,12 @@
 # Deleting All Data for a Metric or a Set of Metrics
 
-This API call is for deleting all of the data from an IRONdb node for a specific metric or for a set of metrics (when wildcards or a tag query are specified). It will remove data for the matching metric(s) throughout all timestamps that have been provided by the user, no matter what the data type.  In addition, it will remove all record of the metric name(s) with their tags and metadata.  This call is intended for removing misnamed/experimental metrics or old metrics which are obsolete and can be safely removed.
+This API call is for deleting all of the data from an IRONdb node for a specific metric or for a set of metrics (when wildcards or a tag query are specified). It will remove data for the matching metric(s) throughout all timestamps and all rollups that have been provided by the user, no matter what the data type.  In addition, it will remove all record of the metric name(s) with their tags and metadata.  This call is intended for removing misnamed/experimental metrics or old metrics which are obsolete and can be safely removed.
 
 When used for deletion of a single metric, this call will return an empty array on success. If there is an error, it will return a JSON object with the error.
 
-When used with wildcards or a tag query, this call always returns a JSON object which describes the matching metrics and the actions taken or errors received on the deletion.  For safety, explicit confirmation is required in the headers to actually force the data deletion.
+When used with wildcards or a tag query, this call always returns a JSON object which describes the matching metrics and the actions taken or errors received on the deletion.
```

Comment:
FYI, line breaks in the markdown document do not translate to HTML `<br>` elements. If you want these lines separate from one another, insert a blank line to make each one its own para.  Wasn't sure your intent here, but I see you've done similar for most of the other pages as well.

Reply:
Yep I know, it has been used for editing convenience, not to force a br, so we are cool